modules/social_features/social_group/modules/social_group_request/social_group_request.services.yml

```diff
@@ -0,0 +1,8 @@
+services:
+
```

Comment:
```suggestion
```

Reply:
I'm using official documentation and and I like the style of the example file there: http://prntscr.com/opwwbo